src/components/deposit.js

```diff
@@ -78,7 +87,20 @@ const Deposit = props => {
     setDepositAllLabel('Deposit all...')
     setStatus('deposit')
 
-    vaultContract.methods.depositAll().send({ from: props.address }).then(() => {
+    let call
+
+    if (vaultContract.methods.depositMATIC) {
+      let amount  = amountToDepositMatic(props.balance.toNumber())
```

Comment:
```suggestion
      let amount = amountToDepositMatic(props.balance.toNumber())
```

And why not as BigNumber?

Reply:
You mean for the function argument? The `value: amount` in the L98 should be just a number but here I can send a BigNumber :+1: